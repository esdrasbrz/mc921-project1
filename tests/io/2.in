if void else char for break
while float assert int
print read return