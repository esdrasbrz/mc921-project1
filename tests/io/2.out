LexToken(IF,'if',1,0)
LexToken(VOID,'void',1,3)
LexToken(ELSE,'else',1,8)
LexToken(CHAR,'char',1,13)
LexToken(FOR,'for',1,18)
LexToken(BREAK,'break',1,22)
LexToken(WHILE,'while',2,28)
LexToken(FLOAT,'float',2,34)
LexToken(ASSERT,'assert',2,40)
LexToken(INT,'int',2,47)
LexToken(PRINT,'print',3,51)
LexToken(READ,'read',3,57)
LexToken(RETURN,'return',3,62)